src/ethereum/prague/fork.py

```diff
@@ -90,13 +90,13 @@
     "0x000F3df6D732807Ef1319fB7B8bB8522d0Beac02"
 )
 HISTORY_STORAGE_ADDRESS = hex_to_address(
-    "0x0aae40965e6800cd9b1f4b05ff21581047e3f91e"
+    "0x0F792be4B0c0cb4DAE440Ef133E90C0eCD48CCCC"
```

Comment:
LGTM so far but will wait to see if the proposed `HISTORY_SERVE_WINDOW` value of `8191` is in fact accepted and the eip PR is merged.

Reply:
Thanks for reviewing this, yes let's see that it gets merged to EIPs first.